src/Illuminate/Validation/Validator.php

```diff
@@ -163,7 +163,6 @@ class Validator implements ValidatorContract
     protected $implicitRules = [
         'Required', 'Filled', 'RequiredWith', 'RequiredWithAll', 'RequiredWithout', 'RequiredWithoutAll',
         'RequiredIf', 'RequiredUnless', 'Accepted', 'Present',
-        // 'Array', 'Boolean', 'Integer', 'Numeric', 'String',
```

Comment:
It was commented by Taylor here: https://github.com/laravel/framework/commit/a7142dc041eb5a5c0507569817f5e450489c5981